doc/articles/features/windows-system.md

```diff
@@ -107,6 +107,35 @@ Exceptions are in line with UWP.
 
 This API is supported on iOS, Android and macOS, and the implementation does not respect the `LaunchQuerySupportType` parameter yet. It also reports the aforementioned special `ms-settings` URIs on Android and iOS as supported.
 
+#### Platform-specific
```

Comment:
My suggested title might not be best, but the idea is to have a more descriptive heading.

```suggestion
#### Platform-specific concerns
```

Reply:
@Youssef1313 I like the idea, but IMO I don't think "concerns" is apt. What about "Platform-specific requirements"? As the <queries> are needed for the feature to work on newer versions?

Reply:
Yes, "requirements" looks good!